index.md

```diff
@@ -16,11 +16,7 @@ The following list provides a summary of the key APIs and features that are used
 
 ## Key features and APIs
 
-**[Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html)** - The Embedding API v3 provides an updated developer experience and enhancements over the JavaScript API v2. The Embedding API v3 provides [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) for embedding Tableau visualizations and is available starting with Tableau version 2021.4. When fully implemented later this year, the Embedding API v3 will support all the functionality of the JavaScript API v2, while providing future enhancements for embedding scenarios.
-
-**[JavaScript API v2](https://help.tableau.com/current/api/js_api/en-us/JavaScriptAPI/js_api.htm)** - an API used on the front-end of the embedding application that initializes Tableau visualizations in JavaScript and provides a wealth of classes and methods driving interactivity between Tableau and your app.
-
-If you are starting a new project, we recommend using the Embedding API v3 as it will receive feature enhancements moving forward.
+**[Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html)** - The Embedding API v3 provides an updated developer experience and enhancements over the JavaScript API v2. The Embedding API v3 provides [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) for embedding Tableau visualizations and is available starting with Tableau version 2021.4. 
```

Comment:
@illonage Just a thought....  If we are going to remove all references to JSAP v2 in this playbook except for this one, we might want to stop comparing v3 to JSAPI v2.  And in that case, we'd probably want rewrite this section slightly.  No need to do it now.

Reply:
you are right, can you take it as an action item?